src/System.Memory/tests/ReadOnlySpan/IndexOfAny.byte.cs

```diff
@@ -80,8 +82,11 @@ public static void DefaultFilledIndexOfTwo_Byte()
                     int index = rnd.Next(0, 2) == 0 ? 0 : 1;
                     byte target0 = targets[index];
                     byte target1 = targets[(index + 1) % 2];
+                    values[0] = target0;
+                    values[1] = target1;
                     int idx = span.IndexOfAny(target0, target1);
                     Assert.Equal(0, idx);
+                    Assert.Equal(idx, span.IndexOfAny(values));
```

Comment:
Nit: it's a bit odd to me that we have two equivalent IndexOfAny calls, but in one case we store into a local and in the other we don't.  I'd prefer to see just:
```C#
Assert.Equal(0, span.IndexOfAny(target0, target1));
Assert.Equal(0, span.IndexOfAny(values));
```
as I think that makes it easier to understand both the correspondence between the two calls, and that for the second one we're explicitly expecting 0 (without my having to understand the previous lines to know that idx == 0).  Plus it's shorter :)